src/frequently-requested-changes.md

```diff
@@ -166,3 +166,15 @@ go anywhere, including in systems contexts where relatively few languages can.
 
 That said, we're happy to add language features to *support* an *optional*
 garbage collector, where needed.
+
+## Suffix modifiers (`if` after `return`/`break`/`continue`, or after arbitrary statements)
+
+We often get proposals for syntax like `return expr if condition;` or
+`break if condition;`. We don't plan to make such a change to Rust.
+
+Such a change would prioritize *concise* code over *readable* code. We don't
```

Comment:
Not sure I necessarily agree with this reasoning. What if it's more important to highlight for the reader the potential for change in control flow?

`if condition { break; }` kind of hides it, and even putting it in an indent on the next line keeps it from standing out if you're reading code in a "breadth first" way, as I often do.

Reply:
This is what I like about `unless`, since it's the general way of saying "this is going to diverge" for those breadth-first readings.  For more see <https://internals.rust-lang.org/t/syntactic-sugar-for-if-x-return-break-continue/19096/14?u=scottmcm>, though I was unsuccessful with advocating for it during `let`-`else`.

Though as someone pointed out on IRLO, it can be done as (the unusual)
```rust
let true = condition else { break };
```

Reply:
@tmandry `return expr if condition` does highlight the `return`, but in doing so it masks the conditional.